lib/src/models/view/schedule.dart

```diff
@@ -2,7 +2,6 @@ import "package:flutter/foundation.dart" show ChangeNotifier;
 
 import "package:ramaz/data.dart";
 import "package:ramaz/models.dart";
-import "package:ramaz/src/data/schedule/schedule.dart";
```

Comment:
make sure to merge the new master then re-push bc I already did this on my pr and don't want to cause a conflict.

Reply:
nvm I took care of it